proj/settings.py

```diff
@@ -9,6 +9,19 @@
     ('Admin', 'admin@example.com'),
 )
 
+DCF_ITEM_MAX_AGE_DAYS = 90
+
+
+EMAIL_HOST = 'smtp.serverhost.com'
```

Comment:
No need to specify EMAIL settings, as it's user responsibility to provide proper EMAIL BACKEND.